includes/Settings/SettingsDebug.php

```diff
@@ -1007,7 +1012,7 @@ public function get_server_config(): array {
 			),
 			'WP Memory Limit' => array(
 				/* translators: <a> tags */
-				'required' => sprintf( __( 'Recommended: 128MB (more for plugin-heavy setups<br/>See: %1$sIncreasing the WordPress Memory Limit%2$s', 'woocommerce-pdf-invoices-packing-slips' ), '<a href="https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/" target="_blank">', '</a>' ),
+				'required' => sprintf( __( 'Recommended: 128MB (more for plugins with heavy setups) See: %1$sIncreasing the WordPress Memory Limit%2$s', 'woocommerce-pdf-invoices-packing-slips' ), '<a href="https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/" target="_blank">', '</a>' ),
```

Comment:
What I meant was to remove it from the string because it’s something that doesn’t need to be translated. Just place it outside the translation function. Right now, it looks strange.

![Screenshot from 2024-12-17 08-23-16](https://github.com/user-attachments/assets/9324905d-202c-4a40-9b67-1b432cf17647)

Reply:
Sorry, I misunderstood your point. Fixed.